spikingjelly/datasets/es_imagenet.py

```
@@ -346,17 +347,18 @@ def create_raw_from_extracted(cls, extract_root: Path, raw_root: Path):
         source_dir = extract_root / "ES-imagenet-0.18" / "val"
         target_dir = raw_root / "test"
         target_dir.mkdir()
-        print(f"Mkdir [{target_dir}].")
+        logger.info("Mkdir [%s].", target_dir)
         utils.create_same_directory_structure(source_train_dir, target_dir)
 
         for fname, label in zip(val_fname, val_label):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)